File: EBSystemBackEnd/EBSystem.Models/ViewModel/Event.cs
namespace EBSystem.Models.ViewModel
{
    public class Event
    {
        public int EventId { get; set; }
        public string? EventName { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public int? EventCategoryId { get; set; }
        public int? TicketCategoryId { get; set; }
        public int? NoOfTickets { get; set; }      
        public string? PromoCode { get; set; }
    }
}

File: EBSystemUI/src/app/Models/event.ts
import { NumberValueAccessor } from "@angular/forms";

export interface Event {
    eventId:number;
    eventName:string;
    startDate:string;
    endDate:string;
    eventCategoryId:number;
    ticketCategoryId:number;
    noOfTickets:number;
    promoCode:string;
}

// "eventId": 1,
// "eventName": "IPL",
// "startDate": "2022-03-09T00:00:00",
// "endDate": "2022-03-22T00:00:00",
// "eventCategoryId": 1,
// "ticketCategoryId": 1,
// "noOfTickets": 1000,
// "promoCode": "IPL0ABT090",
// "event": null,
// "ticketCategory": null

